tests/system/small/test_dataframe.py

```diff
@@ -1705,6 +1700,26 @@ def test_df_stack(scalars_dfs):
     pd.testing.assert_series_equal(bf_result, pd_result, check_dtype=False)
 
 
+def test_df_unstack(scalars_dfs):
+    scalars_df, scalars_pandas_df = scalars_dfs
+    # To match bigquery dataframes
```

Comment:
Didn't quite get this, why can't we use the original `scalars_pandas_df`?

Reply:
So if we remove

```python
scalars_pandas_df = scalars_pandas_df.copy()
```

the following will still apply, just on the original dataframe, no?

```python
scalars_pandas_df.columns = scalars_pandas_df.columns.astype("string[pyarrow]")
```

Reply:
tests share resources, so we create a copy to make sure our mutation doesn't affect other tests

Reply:
Ah, I was missing that, thanks!